### DISCUSSION TITLE
can't login with tenant user in tenant url

### DISCUSSION THREAD

**Discussion by jrpikong** [savannabits/filament-tenancy-starter#4] [General]
Hi @coolsam726 , thanks for sharing, I really appreciate it. 
1. I have done the installation and set up my database with a `mysql`  connection. 
2. I made a tenant from the central dashboard with the name: test1, and I checked has built a new database with the prefix. but not create a user for the tenant database. 
3.  I tried to access http://test1.localhost:3003/login and log in with the user from the tenant database but rejected, but I can login in tenant with the user from central database

please can you help me?

**Comment by coolsam726** [comment]
Hi @jrpikong,
You are welcome.

> I made a tenant from the central dashboard with the name: test1, and I checked has built a new database with the prefix. but not create a user for the tenant database.

Since this is a complete separation of databases, it means that each tenant database has its own data. So yes, the user won't be copied from the central app to the tenant apps.

> I tried to access http://test1.localhost:3003/login and log in with the user from the tenant database but rejected, but I can login in tenant with the user from central database

You have to create a new user (Separate from the central app users) and use them to login to the tenant. The whole idea of database separation is to have complete separation of data between companies. A use case scenario for this setup is when you would like to onboard separate clients into your system using the same code-base, but complete separation so that one client can't access another client's data. That is why this separation is important.
**Hint:** You can create seed the tenants with a default user, or check the `TenantPortalPanelProvider` and enable registration on the panel to allow users to self-register.

**Comment by jrpikong** [comment]
I have done an open register page in the tenant, but I register, the data will be inserted to the central user table, not in the tenant user table

![Screenshot from 2023-11-28 15-58-40](https://github.com/savannabits/filament-tenancy-starter/assets/2731812/df7f9eae-550d-4a1e-97cd-28ea29989e51)

**Reply by coolsam726** [reply]
If your setup is correct the data should be inserted into the tenant database, not the central database (when you are serving from the tenant subdomain)

**Reply by jrpikong** [reply]
I just cloned, ran docker, and ran the migration from the central app.
Is there anything I need to do besides that?
